sources/packages/api/src-sql/ApplicationNumber/Create-application-number.sql

```diff
@@ -0,0 +1,7 @@
+
+-- Add the column application_number for application.
+ALTER TABLE applications ADD 
+    COLUMN IF NOT EXISTS application_number VARCHAR(10);

```

Comment:
IMO, a length shorter than 10 would be inconsistent. I believe that the type should be CHAR(10) instead.